docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
```

Comment:
Ensure that the `.env` file contains all necessary environment variables for both the Django application and the PostgreSQL database. This is crucial for the services to function correctly.